transforms/language/doc_chunk/transform.config

```diff
@@ -14,7 +14,7 @@ TRANSFORM_NAME=doc_chunk
 #
 # If you change the versions numbers, be sure to run "make set-versions" to 
 # update version numbers across the transform (e.g., pyproject.toml).
-DOC_CHUNK_PYTHON_VERSION=$(DPK_VERSION)
+DOC_CHUNK_PYTHON_VERSION=0.3.0
```

Comment:
While I understand he wants to set his own versions, how do we update this (not by hand) when we do a major upgrade.  Do we need 2 `set-version` targets or will `set-versions` update this?

Reply:
We may need to traverse all the transform.config and bump up the last digit or append .devX to it when need to.

Reply:
ok, kick the can down the road :)